### DISCUSSION TITLE
Popups not showing

### DISCUSSION THREAD

**Discussion by ESTgrrt** [Tunny-gh/Tunny_in_Open_Sea#200] [General]
Once it finishes the optimization the popup telling me that it is done does not show up, it gets stuck with the green bar almost at the end and the study doesn't save. This started happening after I updated rhino I believe.

![Screenshot 2023-09-19 121804](https://github.com/hrntsm/Tunny/assets/97501557/ee07e56b-3991-43b0-abd4-4d68a430cb3e)

**Comment by hrntsm** [comment]
Hi @ESTgrrt,

Sometimes the finish window is hidden behind the Rhino window.
Can you mouse over the Rhino icon to see if the finish window appears?

<img width="594" alt="Screenshot 2023-09-20 at 9 07 21" src="https://github.com/hrntsm/Tunny/assets/23289252/e120d89b-c94f-4474-a0fb-d6049d3febea">

**Reply by ESTgrrt** [reply]
Thanks for the reply! yeah I've seen that happen a couple of times but in this case that wasn't happening, doing a complete reinstalled fixed the issue as it also wasn't letting me see my trials. Turns out when I check InMemory it kind of bugs out and it doesn't show the popus, but I don't know if that's on my end due to my system capabilities. Without InMemory it runs smoothly.

**Reply by hrntsm** [reply]
Perhaps the saving of the optimization results is failing.
The following link may help if you explicitly specify the path to the file where you want to save the results.

https://tunny-docs.deno.dev/docs/optimize-window/file-tab
